activate

```diff
@@ -71,25 +82,45 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 # detect live system
 live="false"
 if grep -qw "endless.live" /proc/cmdline; then
-    image="true"
+    live="true"
+fi
+
+
+# We don't want dummy serials sent
+if [ "${serial}" == "To be filled by O.E.M." ]; then
```

Comment:
Serial is the important one so we don't reprocess the same data over and over again including sending to asus but if that's not an issue, I'll take that out.